cl/audio/management/commands/transcribe.py

```diff
@@ -61,12 +61,12 @@ def handle_open_ai_transcriptions(options) -> None:
     """
     requests_per_minute = options["rpm"]
 
-    if options.get("retry_failures"):
-        query = Q(stt_status=Audio.STT_FAILED)
-    else:
-        query = Q(stt_status=Audio.STT_NEEDED)
+    status_code = options.get("status_to_process")
+    for code, descr in Audio.STT_STATUSES:
+        if code == status_code:
+            logger.info("Querying Audio.stt_status = %s. %s", code, descr)
 
```

Comment:
This feels like a lot of code just for a better `desc` in the log line.

Reply:
It made sense to work with the integer values, because they are exposed in the Audio class and straightforward to validate, but it makes the user experience worse. Not the best looking, but we can alleviate that a little with these lines